option/query.go

```diff
@@ -37,37 +29,40 @@ func WithParameter(name, valueType, value string) QueryOption {
 	}
 }
 
-// WithAsyncClientTimeout is maximum amount of time that the client is willing to wait for the query to complete.
-// If the query is not complete by this timeout, a response will be returned with a query_id that can be used to
-// check the status of the query and retrieve results once the query has completed.
-func WithAsyncClientTimeout(timeout int64) QueryOption {
+// WithTimeout maximum amount of time that Rockset will attempt to complete query execution before
+// aborting the query and returning an error. The query timeout defaults to a maximum of 2 minutes. 
+// If WithAsync is set, the query timeout defaults to a maximum of 30 minutes.
```

Comment:
Go documentation comments must start with the function name, i.e. 
```
// WithAsync ...
```

Is there ever a case when you explicitly want to set `WithAsync(false)`? If not, the function signature can just  be
```
func WithAsync() QueryOption
````